src/renderer/components/icon/icon.tsx

```diff
@@ -121,7 +121,7 @@ export class Icon extends React.PureComponent<IconProps> {
     }
 
     // render as material-icon
-    if (material) {
+    if (material && typeof material === "string") {
```

Comment:
Why is this check needed? Isn't the type of `material` string or undefined so it should always be a string?

Reply:
yeah, if users are using typescript (some just use javascript). I have tried to mock a category which has `category.metadata.icon = {}` and managed to pass `{}` to material props here (and crash the component) 

https://github.com/lensapp/lens/blob/da3c8d00c7ebe96c67b8c1be09878b4234ba6bf6/src/renderer/components/%2Bcatalog/catalog-menu.tsx#L48

Reply:
shouldn't you also do the check for `svg`? And wouldn't it be good enough to just do a `typeof` check?

Reply:
That's true, thanks.

Reply:
Added https://github.com/lensapp/lens/pull/3927/commits/c55b2fa1ae0cec757985c30b2bf4f8058b1ed9f3